Doc/library/random.rst

```diff
@@ -111,6 +111,9 @@ Bookkeeping functions
    as an optional part of the API. When available, :meth:`getrandbits` enables
    :meth:`randrange` to handle arbitrarily large ranges.
```

Comment:
I would prefer to explicitly document getrandbits(0) special case: mention that it's not random, but always return 0. Since it contradicts the "random bits" doc above.

Reply:
getrandbits(0) is not more special than randrange(1, 2), uniform(10, 10) or choice([7]).

Reply:
Ok, ignore my comment. Serhiy is right. First, I documented randbytes(0) special case in my PR. But after Serhiy explained that other methods can also be deterministic, I removed the sentence.